src/work-allocation-2/containers/task-list-wrapper/task-list-wrapper.component.ts

```diff
@@ -139,13 +146,13 @@ export class TaskListWrapperComponent implements OnInit {
         order: this.taskServiceConfig.defaultSortDirection
       };
     }
-
+    const pageSorted = +this.sessionStorageService.getItem(this.pageSessionKey);
     this.isPaginationEnabled$.subscribe({
```

Comment:
isPaginationEnabled should not be there for release 2. It should always be enabled

Reply:
Done